css/at-rules.json

```diff
@@ -54,7 +54,7 @@
       "prefix": {
         "syntax": "<symbol>",
         "media": "all",
-        "initial": "\"\"",
```

Comment:
I believe it should be `"\"\""` since it's a valid CSS value

Reply:
I changed this so it will place nicely with l10n check. Can bring it back if you want?